packages/playwright/src/common/config.ts

```diff
@@ -20,6 +20,7 @@ import os from 'os';
 import type { Config, Fixtures, Project, ReporterDescription } from '../../types/test';
 import type { Location } from '../../types/testReporter';
 import type { TestRunnerPluginRegistration } from '../plugins';
+import { gitCommitInfo } from '../plugins/gitCommitInfoPlugin';
```

Comment:
I'm gettin linter error on this line:
```
Checking DEPS for html-reporter
Checking DEPS for playwright-ct-core
Disallowed import ../playwright/src/plugins/gitCommitInfoPlugin.ts in ../playwright/src/common/config.ts
```
How to fix it?

Reply:
You can add new dependencies to the `DEPS.list` file in the same folder, if it's a legit dependency. In case of plugins, it doesn't seem right that `common` depends on `plugins`. Take a look at `webServerPluginsForConfig`, something similar should work for git info.